rubicon_ml/intake_rubicon/publish.py

```diff
@@ -118,5 +122,24 @@ def _build_catalog(experiments):
 
         experiment_catalog_name = f"experiment_{experiment.id.replace('-', '_')}"
         catalog["sources"][experiment_catalog_name] = appended_experiment_catalog
+    
+    # create visualization entry to the catalog file
+    if visualization != None:
+        appended_visualization_catalog = {
+            "driver": "rubicon_ml_experiment_table",
+            "args": {
+                "experiments": visualization.experiments,
```

Comment:
we won't need the experiments themselves. we'll assume they're present as actual experiment sources in the catalog too, and later add some checks to validate that

Reply:
Okay I'll remove experiments from it.